chrome/background.js

```diff
@@ -401,11 +401,21 @@ passwordalert.background.injectContentScriptIntoAllTabs_ =
   chrome.tabs.query({}, function(tabs) {
     for (var i = 0; i < tabs.length; i++) {
       var tabIdentifier = tabs[i].id + ' - ' + tabs[i].url;
+
+      // Skip any pages not starting with https://
```

Comment:
I don't think we'd actually want to restrict to https:// nor https://www. as the code below appears to do.

I can't reproduce the issue at https://github.com/google/password-alert/issues/91 so there's probably nothing we need to do for this.

Reply:
sure!